src/java.base/share/classes/java/lang/reflect/code/bytecode/BytecodeGenerator.java

```diff
@@ -860,8 +860,27 @@ private void generate() {
                         push(op.result());
                     }
                     case InvokeOp op -> {
-                        // @@@ var args
-                        processOperands(op);
+                        if (op.isVarArgs()) {
+                            processOperands(op.operands().subList(0, op.operands().size() - op.varArgOperands().size()));
```

Comment:
Suggest we add the compliment to `varArgOperands()`, called say `argOperands()` such that `operands().equals(<argOperands() + varArgOperands()>)`